packages/v4/src/content/design-guidelines/components/card/card.md

```diff
@@ -3,116 +3,113 @@ id: Card
 section: components
 ---
 
+
 A **card** is a square or rectangular container that can contain any kind of content. Cards symbolize units of information, and each one acts as an entry point for users to access more details. For example, in dashboards and catalog views, cards function as a preview of a detailed page. Cards may also be used in data displays like card views, or for positioning content on a page.  
 
 ## Elements
 
-A card usually consists of 3 parts: 
+A card usually consists of four parts:
+
+<img src="./img/card.png" alt="4 parts of a card" width="394"/>  
+
+
+1. **Header (optional):** When included, defines the contents of a card. Card headers can contain images as well as the title of a card and an actions menu represented by the right-aligned kabob.
+
+2. **Title (optional):** Communicates the title of a card if it's not included in the header.
+
+3. **Body:** Provides details about the item. A card body can include any combination of static text and/or active content.
+
+4. **Footer (optional):** Contains links, actions, or static text at the bottom of a card.
+
+Cards are flexible and customizable to your use case. Here are some different ways to use a card to represent the same content. Choose a layout that considers the usage context and the space you have to work with.
+
+<img src="./img/card-variants.png" alt="three card variants" width="1202"/>
 
-<img src="./img/card.png" alt="3 parts of a card" width="394"/>  
+1. The first version places an image in the header with a separate title. The View details action is located in the footer.
```

Comment:
How do you feel about bolding the words "View details" to separate them as an action?